langchain4j-weaviate/src/test/java/dev/langchain4j/store/embedding/weaviate/LocalGRPCWeaviateEmbeddingStoreIT.java

```diff
@@ -0,0 +1,74 @@
+package dev.langchain4j.store.embedding.weaviate;
+
+import dev.langchain4j.data.segment.TextSegment;
+import static dev.langchain4j.internal.Utils.randomUUID;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import java.util.Arrays;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.weaviate.WeaviateContainer;
+
+@Testcontainers
+class LocalGRPCWeaviateEmbeddingStoreIT extends EmbeddingStoreIT {
+
+    @Container
+    static WeaviateContainer weaviate = new WeaviateContainer("semitechnologies/weaviate:latest")
+            .withEnv("QUERY_DEFAULTS_LIMIT", "25")
+            .withEnv("DEFAULT_VECTORIZER_MODULE", "none")
+            .withEnv("AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED", "true")
```

Comment:
is `AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED` required? seems to work fine without it